smooth_swing_v2.h

```diff
@@ -0,0 +1,259 @@
+#ifndef SOUND_SMOOTH_SWING_V2_H
```

Comment:
Seems like this file has the wrong name?

Reply:
uploaded into the wrong directory